lib/logflare/application.ex

```diff
@@ -181,6 +181,21 @@ defmodule Logflare.Application do
            start_pool_metrics?: true
          ]
        }},
+      {Finch,
+       name: Logflare.FinchBQStorage,
+       pools: %{
+         "https://bigquerystorage.googleapis.com" => [
+           protocols: [:http2],
+           count: max(base, 20) * 2,
```

Comment:
```suggestion
           count: max(base, 20), 
```
Can remove the x2 first and observe pool metrics before we bump this up more